### PR TITLE
OSSMDOC-588: Distributed tracing 2.5 Release Notes.

### PR DESCRIPTION
**Pull request by JStickler**
Version(s): 4.6 - 4.12

Issue: [OSSMDOC-588](https://issues.redhat.com/browse/OSSMDOC-588)

Link to docs preview: http://file.bos.redhat.com/jstickle/OSSMDOC-588/distr_tracing/distributed-tracing-release-notes.html#distr-tracing-rn-new-features_distributed-tracing-release-notes

Additional information:
Eng review: pavolloffay
QE review:  iblancasa
Peer review: tmalove

### CODE REVIEW

**Review by pavolloffay — APPROVED** [review]

File: modules/distr-tracing-rn-new-features.adoc [hunk 1]
```diff
@@ -63,7 +89,7 @@ This release of {DTProductName} addresses Common Vulnerabilities and Exposures (
 
 This release of {DTProductName} addresses Common Vulnerabilities and Exposures (CVEs) and bug fixes.
 
-With this release, the {JaegerName} Operator is now installed to the `openshift-distributed-tracing` namespace by default.  Previously the default installation had been in the `openshift-operator` namespace.
+With this release, the {JaegerName} Operator is now installed to the `openshift-distributed-tracing` namespace by default. Previously the default installation had been in the `openshift-operator` namespace.
```
Comment (iblancasa) [inline modules/distr-tracing-rn-new-features.adoc:92]:
```suggestion
With this release, the {JaegerName} Operator is now installed to the `openshift-distributed-tracing` namespace by default. Previously the default installation had been in the `openshift-operators` namespace.
```

Comment (iblancasa) [inline modules/distr-tracing-rn-new-features.adoc:92]: If somebody removes a Jaeger Operator running in `openshift-operators`, let Jaeger instances running and then installs a new version of the Jaeger Operator in `openshift-distributed-tracing`, the Jaeger instances will not be upgraded. Should we document that?

**Review by iblancasa — CHANGES_REQUESTED** [review]

File: modules/distr-tracing-rn-new-features.adoc (hunk 1 above)
Comment (pavolloffay) [reply modules/distr-tracing-rn-new-features.adoc:92]: It does not seem like a common use-case to uninstall the operator while there are running instances.

Comment (JStickler) [reply modules/distr-tracing-rn-new-features.adoc:92]: I don't even know if it's possible to remove an Operator with instances still running.

Comment (JStickler) [reply modules/distr-tracing-rn-new-features.adoc]: Good catch, fixed!

Comment (iblancasa) [reply modules/distr-tracing-rn-new-features.adoc]: @JStickler it is.

**Review by iblancasa — APPROVED** [review]

File: modules/distr-tracing-rn-new-features.adoc (hunk 1 above)
Comment (JStickler) [reply modules/distr-tracing-rn-new-features.adoc]: @iblancasa That feels like a usability bug to me.

**Review by tmalove — COMMENTED** [review]
LGTM

**Pull request merged**